app/src/main/AndroidManifest.xml

```diff
@@ -29,7 +29,7 @@
 
         <provider
             android:name="androidx.core.content.FileProvider"
-            android:authorities="com.randomappsinc.studentpicker.provider"
+            android:authorities="${applicationId}.provider"
```

Comment:
@Gear61 
changed this to use (com.randomappsinc.studentpicker) instead of (com.randomappsinc.studentpicker.debug)

Reply:
![Screenshot_20200424-005734](https://user-images.githubusercontent.com/25705658/80153869-0b223a00-85c7-11ea-8b31-ff281cb5a146.png)